src/components/App/App.js

```diff
@@ -25,16 +25,16 @@ class App extends Component {
 
   componentWillMount() {
     const baseUrl = window.location.href.replace(`#${this.props.location.pathname}`, '');
-    this.props.store.base_path = baseUrl;
+    this.props.store.addPath(baseUrl);
 
-    this.props.store.components = window.sources.reduce((acc, val) => {
+    const components = window.sources.reduce((acc, val) => {
       const slug = val.split('/')[val.split('/').length - 1];
       const content = this.getMarkup(val, slug);
       const config = yaml.load(this.fixPath(`${val}/${slug}.yml`));
-      const variants = config && config.variants ? Object.keys(config.variants).map((key) => {
+      const variants = config && config.variants ? config.variants.map((key) => {
         return {
           slug: key,
-          title: config.variants[key],
+          title: key,
```

Comment:
🤔 Yes, but here we are talking about variant's title. It seems weird to repeat the same title on each variant 😅